content/fr/docs/concepts/overview/working-with-objects/namespaces.md

```diff
@@ -0,0 +1,116 @@
+---
+reviewers:
+- derekwaynecarr
+- mikedanese
+- thockin
```

Comment:
```suggestion
```

Reply:
Those people probably don't speak French and do not need to be included here. They only review the English version